src/_includes/components/funders.njk

```diff
@@ -8,7 +8,7 @@
 </svg>
 
 <div class="funders">
-	<p>Funded by a grant from <a class="hewlett-href" href="https://hewlett.org/">The William and Flora Hewlett Foundation</a>.</p>
+	<p>Funded by a grant from The William and Flora Hewlett Foundation.</p>
```

Comment:
Please remove the css definition for `.hewlett-href` too. Thanks.

Reply:
@cindyli accounted for that already. I already did.

Reply:
Yes, you did. Sorry I missed those lines. Thanks.

Reply:
Look at the review and see line 53 in` _footer.scss`